data/virt_autotest/host_unattended_installation_files/agama/sle16.jsonnet

```diff
@@ -35,9 +35,12 @@
   },
   software: {
       patterns: [
-         'base',
-         'kvm_server',
-         'kvm_tools'
+        'base',
+        'kvm_server',
+        'kvm_tools'
+      ],
+      packages: [
+        'virt-bridge-setup'
```

Comment:
Will it automatically run to setup br0 at reboot? If yes, maybe let's hold on. Otherwise good.

Reply:
No, it just installs the package. "br0" can be set up by running the tool seperately.